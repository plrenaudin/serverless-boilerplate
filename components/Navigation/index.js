import "isomorphic-unfetch";
import Link from "next/link";
import "./index.less";

const Navigation = () => (
  <nav className="mainNavigation">
    <Link href="/">
      <a>Home</a>
    </Link>
    <Link href="/about">
      <a>About</a>
    </Link>
  </nav>
);

export default Navigation;
