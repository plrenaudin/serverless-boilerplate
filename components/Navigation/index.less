@import "../../styles/common.less";

.mainNavigation {
  a {
    padding: 0 @spacing;
  }
}
