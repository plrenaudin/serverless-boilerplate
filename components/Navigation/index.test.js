import Navigation from ".";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const tree = renderer.create(<Navigation />).toJSON();
  expect(tree).toMatchSnapshot();
});
