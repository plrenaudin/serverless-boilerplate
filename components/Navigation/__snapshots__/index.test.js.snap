// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  className="mainNavigation"
>
  <a
    href="/"
    onClick={[Function]}
    onMouseEnter={[Function]}
  >
    Home
  </a>
  <a
    href="/about"
    onClick={[Function]}
    onMouseEnter={[Function]}
  >
    About
  </a>
</nav>
`;
